test/testRunner.ts

```diff
@@ -79,17 +79,6 @@ async function main(): Promise<void> {
     fs.mkdirSync(path.dirname(settings_dst), { recursive: true });
     fs.copyFileSync(settings_src, settings_dst);
 
-    // Install the ansible extension from generated .vsix
```

Comment:
With the change on `extensionDevelopmentPath` below, we no longer need to install .vsix separately.